subgraph.mantle.yaml

```
@@ -86,3 +86,28 @@ dataSources:
+      abis:
+        - name: SingleRequestProxyFactory
+          file: ./abis/SingleRequestProxyFactory.json
+      eventHandlers:
+        - event: ERC20SingleRequestProxyCreated(address,address,address,indexed bytes,address,uint256,address)
+          handler: handleERC20SingleRequestProxyCreated
+          receipt: true
+        - event: EthereumSingleRequestProxyCreated(address,address,indexed bytes,address,uint256,address)
+          handler: handleEthereumSingleRequestProxyCreated
+          receipt: true
+      file: ./src/singleRequestProxyFactory.ts
+  
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)